core/build.gradle

```diff
@@ -52,10 +50,16 @@ dependencies {
     implementation(group: "com.jayway.jsonpath", name: "json-path", version: json_path_version) { exclude(module: "json-smart"); exclude(group: "org.slf4j") }  // Apache 2.0
     implementation group: "org.codehaus.janino", name: "janino", version: janino_version  // BSD
     implementation group: "org.codehaus.janino", name: "commons-compiler", version: commons_compiler_version  // BSD
-    implementation group: 'com.j256.simplemagic', name: 'simplemagic', version: simplemagic_version // ISC
+    implementation group: "com.j256.simplemagic", name: "simplemagic", version: simplemagic_version // ISC
 
-    api(group: "org.apache.calcite", name: "calcite-linq4j", version: calcite_linq4j_version) { exclude(module: "guava") } // Apache 2.0
-    api(group: "org.polypheny.avatica", name: "avatica-core", version: avatica_core_version) { exclude(module: "protobuf-java") } // Apache 2.0
+    api(group: "org.apache.calcite", name: "calcite-linq4j", version: calcite_linq4j_version) {
+        exclude(module: "guava")
+        exclude(module: "avatica-core")
+    } // Apache 2.0
+    api(group: "org.polypheny.avatica", name: "avatica-core", version: avatica_core_version) {
+        exclude(module: "protobuf-java")
+        exclude group: "org.apache.calcite", module: "avatica-core"
+    } // Apache 2.0
```

Comment:
Can we try to already get rid of Avatica in the core module, or shall we tackle this in the next PR?

Reply:
Sadly not, it is still extensively used in some classes like, PolyphenyDbConnectionConfig etc.